<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathOperationClass</class>
 <widget class="QWidget" name="MathOperationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MathOperation</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>1191</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView_1"/>
    </item>
    <item>
     <widget class="QLabel" name="op">
      <property name="text">
       <string>op</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView_3"/>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="plus_slider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_image1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_image2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="plus_rb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="minus_rb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="multiply_rb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="divide_rb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QSlider" name="divide_slider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>-250</number>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="not_rb">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>！</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="or_rb">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="and_rb">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;&amp;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="xor_rb">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="scale_rb">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>440</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>缩放</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="verflip_rb">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>410</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>垂直翻转</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="horiflip_rb">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>380</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>水平翻转</string>
   </property>
  </widget>
  <widget class="QSlider" name="scale_slider">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>440</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>-2</number>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>α：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="alpha">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>β：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="beta">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>C：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Constant">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>440</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="times">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Reset_btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>请先选择图1</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MathOperation.qrc"/>
 </resources>
 <connections/>
</ui>
